backend/src/models/Student.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose')
+
+const StudentSchema = new mongoose.Schema(
+    {
+        // create a new schema for the student
+        name: {
+            type: String,
+            required: true,
+            minlength: 3,
+            maxlength: 30,
+        },
+        email: {
+            type: String,
+            required: true,
+            minlength: 6,
+            maxlength: 255,
+        },
+        icard_number: {
+            type: String,
+            required: true,
+            minlength: 6,
+            maxlength: 10,
+        },
+        active: {
+            type: Boolean,
+            required: true,
+        },
+        isaf_paying_status: {
+            type: Boolean,
+            required: true,
+        },
+    },
+    { collection: 'students', versionKey: false }
+) // set the collection name to temp, this is just for testing
```

Comment:
is this still a relevant comment? it's no longer named temp right

Reply:
oh, I left it there by mistake. I have removed it